skimage/_build.py

```diff
@@ -38,6 +39,10 @@ def cython(pyx_files, working_path=''):
             if not _changed(pyxfile):
                 continue
 
+            if pyxfile.endswith('.pyx.in'):
+                process_tempita_pyx(pyxfile)
+                pyxfile = pyxfile.replace('.pyx.in', '.pyx')
```

Comment:
Most of Cythom/Tempita stuff is from scipy's `cythonize.py` script, which itself quotes Dag Sverre Seljebotn